src/main/java/components/MainWindow.java

```diff
@@ -0,0 +1,57 @@
+package components;
+
+import java.util.Objects;
+
+import henry.Henry;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+
+/**
+ * Controller for MainWindow. Provides the layout for the other controls.
+ */
+public class MainWindow extends AnchorPane {
+    @FXML
+    private ScrollPane scrollPane;
+    @FXML
+    private VBox dialogContainer;
+    @FXML
+    private TextField userInput;
+    @FXML
+    private Button sendButton;
+
+    private Henry henry;
+
+    private final Image userImage = new Image(
+        Objects.requireNonNull(this.getClass().getResourceAsStream("/images/user.png")));
+    private final Image henryImage = new Image(
+        Objects.requireNonNull(this.getClass().getResourceAsStream("/images/henry.png")));
+
+    @FXML
+    public void initialize() {
```

Comment:
Not sure if you would like to add JavaDocs for this method too

Reply:
Will consider for future commit.